client-core/Cargo.toml

```diff
@@ -15,7 +15,11 @@ chain-tx-validation = { path = "../chain-tx-validation" }
 chain-storage = { path = "../chain-storage", default-features = false }
 once_cell = "1.4"
 mock-utils = { path = "../chain-tx-enclave/mock-utils" }
-ledger-crypto = { git = "https://github.com/Zondax/ledger-crypto-rs", rev = "62ccadf0374c45e318e622cc01fb44647592659f" }
+ledger-crypto = { git = "https://github.com/linfeng-crypto/ledger-crypto-rs", rev = "3e0a2dadc52be437c44b1ad5a0691fd0337bcac3" }
+ledger-transport-hid = { git = "https://github.com/linfeng-crypto/ledger-rs", rev = "8d94ced5cdd9dc3255bf0e6e775e4ec397b61f0d" }
+ledger-transport-zemu = { git = "https://github.com/linfeng-crypto/ledger-rs", rev = "8d94ced5cdd9dc3255bf0e6e775e4ec397b61f0d" }
+ledger-transport = { git = "https://github.com/linfeng-crypto/ledger-rs", rev = "8d94ced5cdd9dc3255bf0e6e775e4ec397b61f0d" }
```

Comment:
forked it here: https://github.com/crypto-com/ledger-rs/tree/zemu
so can point to it before it's merged in main repo, it can point to that instead

Reply:
the ledger-crypto-rs also need to be forked.

Reply:
https://github.com/crypto-com/ledger-crypto-rs